src/KafkaFlow/Configuration/ClusterConfiguration.cs

```diff
@@ -12,6 +12,7 @@ public class ClusterConfiguration
         private readonly Func<SecurityInformation> securityInformationHandler;
         private readonly List<IProducerConfiguration> producers = new();
         private readonly List<IConsumerConfiguration> consumers = new();
+        private readonly List<TopicConfiguration> topicsToCreateIfNotExist;
```

Comment:
It could be an IEnumerable since it is not modified during the object's life cycle...

Reply:
We want to expose it as ReadOnlyCollection, so your suggestion made me realize that we could do the conversion right away. What do you think?